schemas/event.yml

```diff
@@ -38,53 +38,174 @@
-      example: event
-      accepted_values:
-      - name: alert
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
-      - name: event
-        description: >
-          Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
-          incididunt ut labore et dolore magna aliqua.
+        This is one of four ECS Categorization Fields, and indicates the
+        highest level in the ECS category hierarchy.
+
+        `event.kind` gives high-level information about what type of information the
+        event contains, without being specific to the contents of the event.
+        For example, values of this field distinguish alert events from
+        metric events.
+      example: alert
+      allowed_values:
+        - name: alert
+          description: >
+            `event.kind:alert` indicates an event that describes an alert.
+            Alerts are often associated with detection rules. `event.kind:alert`
+            is often populated for events coming from firewalls, intrusion detection
+            systems, endpoint detection and response systems, etc.
+            It is used to indicate that an alert was triggered.
+        - name: event
+          description: >
+            `event.kind:event` is the most general and most common value of this
+            field. It is used to represent events that indicate that something happened.
+        - name: metric
+          description: >
+            Used to indicate that this event is a measurement taken at given point in time.
+            Examples include CPU utilization, memory usage, or a vulnerability scan result.
+            Events with `event.kind:metric` indicate that a measurement was taken.
+        - name: state
+          description: >
+            This value is similar to metric, except that the entity being measured does not
+            provide a numeric metric value, but rather one of a fixed set of conditions or states.
+            For example a periodic event reporting a "fin_wait" state of a TCP connection
+            on a host might use `event.type:state`.
+        - name: pipeline_error
+          description: >
+            This value indicates that an error occurred during the ingestion of this event,
+            and that event data may be missing, inconsistent, or incorrect.
+            `event.kind:pipeline_error` is often associated with parsing errors.
+        - name: signal
+          description: >
+            The signal value is used by Elastic Kibana apps, such as SIEM, for app-specific purposes.
+            `event.kind:signal` is thus reserved and should not be used for the ingestion
+            of events into Elasticsearch.
 
     - name: category
       level: core
       type: keyword
-      short: Event category.
+      short: Event category. (Categorization Field)
```

Comment:
```suggestion
      short: Event category. The second categorization field in the hierarchy.
```

Reply:
Can we make this consistent with the `kind` and `outcome` fields, see sentence structure:

> The outcome of an event.

v.

> Event category.

Reply:
I'm for consistency, but I think we should also strive to describe fields without using the name of a field.
Neither "The outcome of an event", nor "the event's outcome" tell me what an _outcome_ actually is.

Similarly, telling its position in the hierarchy doesn't help much. I think need to link to a separate ECS hierarchy page/doc from here, where we can explain more about what the hierarchy is and how it works.

Reply:
The shortness of the short definition make it challenging to keep good style practices, but I agree we can do better. I vote for keeping the hierarchy comments for now, but I love the idea of a diagram to be added later.